src/App.css

```diff
@@ -5,244 +5,335 @@
+    position: absolute;
+    bottom: 0;
+    left: 0;
+}
+
+
+
+.movingGradient {
+    /* border: 1px solid black; */
+    border-radius: 50px;
+    position: relative;
+    /* left: 25%;
+    right: 25%; */
+    height: 40%;
+    width: 40%;
+    margin: 0 auto;
+    margin-top: 5%;
+    text-align: center;
+    z-index: 98;
 }
 
 .playContainer {
-  border: 4px solid black;
-  border-radius: 50px;
-  position: relative;
-  /* left: 25%;
-  right: 25%; */
-  height: 40%;
-  width: 40%;
-  margin: 0 auto;
-  margin-top: 5%;
-  text-align: center;
-  background: rgba(107, 123, 50, 0.5);
+    border: 2px solid black;
+    /* border-radius: 50px; */
+    position: absolute;
+    /* left: 25%;
+    right: 25%; */
+    height: 100%;
+    width: 100%;
+    margin: 0 auto;
+    text-align: center;
+    /* background: rgba(107, 123, 50, 1); */
+    background: #9CCC9C;
+
+    display: block;
+    z-index: 99;
+}
+.movingGradient:before,
+		.movingGradient:after{
+			content: '';
+			position: absolute;
+			top: -3px;
+			left: -3px;
+			width: calc(100% + 6px);
+			height: calc(100% + 6px);
+            background: linear-gradient(45deg, rgb(81, 189, 252), rgb(69, 253, 177), rgb(81, 149, 252), rgb(101, 248, 224), rgb(99, 235, 128), rgb(43, 240, 125));
+			/* background: linear-gradient(45deg, red, yellow, green, blue, yellow, purple); */
+			z-index: -1;
+			background-size: 400%;
+			animation: borderbg 40s linear infinite;
+		}
+		@keyframes borderbg {
+			0%{
+				background-position: 0 0 ;
+			}
+			50%{
+				background-position: 400% 0;
+			}
+			100%{
+				background-position: 0 0 ;	
+			}
+		}
+		.movingGradient:after{
+			filter: blur(40px);
+		}
+
+.testBackground {
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+    background-color: #000;
+    font-family: Open Sans, sans-serif;
 }
 
+
 .skins-button {
-  color: #000;
-  font-size: 4rem;
-  position: absolute;
-  top: 10%;
-  left: 10%;
+    color: rgb(0, 0, 0);
+    font-size: 4rem;
+    position: absolute;
+    top: 10%;
+    left: 10%;
 }
 
 .gameplay-button {
-  color: #000;
-  font-size: 4rem;
-  position: absolute;
-  bottom: 10%;
-  left: 45%;
+    color: rgb(0, 0, 0);
+    font-size: 4rem;
+    position: relative;
+    top: 60px;
 }
 
 .upgrades-button {
-  color: #000;
-  font-size: 4rem;
-  position: absolute;
-  top: 10%;
-  right: 10%;
+    color: rgb(0, 0, 0);
+    font-size: 4rem;
+    position: absolute;
+    top: 10%;
+    right: 10%;
 }
 
 .userInput {
-  position: absolute;
-  top: 35%;
-  left: 33%;
+    color: rgb(80, 202, 86);
+    position: absolute;
+    top: 45%;
+    width: 100%;
 }
 
 .inputText {
-  font-family: 'Quicksand', sans-serif;
-  font-size: 150%;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 150%;
 }
 
 .inputBox {
-  font-family: 'Quicksand', sans-serif;
-  font-size: 150%;
-  border-radius: 15px;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 150%;
+    border-radius: 15px;
 }
 
 
```

Comment:
Instead of using the Image Slider, we now use the horizontal menu dependency for our project. Consider remove CSS Classes related to the Image Slider.